docs/deployments/oai-nfapi-sim-compose/README.md

```diff
@@ -1,20 +1,28 @@
 # OpenAir Core + OpenAir RAN as a docker-compose
-We will use 5G L2 nFAPI simulator to test L2 and above Layers. Let's pull Eurecom's deployment for 5G SA mode with 1 User: [OAI Full Stack 5G-NR L2 simulation with containers and a proxy](https://gitlab.eurecom.fr/oai/openairinterface5g/-/tree/develop/ci-scripts/yaml_files/5g_l2sim_tdd) and replace the UPF with our eUPF.
+We will use 5G L2 nFAPI simulator to test L2 and above Layers. Let's pull Eurecom's deployment for 5G SA mode with 1 User: [OAI Full Stack 5G-NR L2 simulation with containers and a proxy](https://gitlab.eurecom.fr/oai/openairinterface5g/-/tree/f1d080d31515d3cc9a31a41fd548753ccba4f09f/ci-scripts/yaml_files/5g_l2sim_tdd) and replace the UPF with our eUPF.

```

Comment:
It's better to use tags instead of direct commit hash

Reply:
Yes, tags is better.  But here we have faulty releases and configs. That commint found as working, and it It is parent to commit with faulty changes to docker-compose and config files. See Troubleshooting section in https://github.com/edgecomllc/eupf/issues/547#issuecomment-2123273170